### DISCUSSION TITLE
Allow realigning of images that are displayed in-line?

### DISCUSSION THREAD

**Discussion by evanscybersecurity** [standardnotes/forum#3329] [Suggestions]
When I cut and paste an image in to a note it automatically places it in the center when the text is set to left justified. 

Is there a way to change the location of that image? I want to have it over to the left so that it lines up with the text.

Thanks

**Comment by effieeee** [comment]
Hi @uYtLD9zS, I'm afraid it's also not possible to realign images that are displayed in the notes, though as with your [other concern](https://github.com/standardnotes/forum/discussions/3328), this has been converted into a suggestion 🙏

**Comment by evanscybersecurity** [comment]
Is there a different way that I can bring in an image and have it on the left side?

**Reply by effieeee** [reply]
If you use a **Rich Text** note instead, you should be able to insert and align an image via HTML, but the image must be linked via a publicly accessible URL.

**Reply by evanscybersecurity** [reply]
Thank you for your response
